spk/demoservice/src/conf/resource

```diff
@@ -0,0 +1,17 @@
+{
+  "mysql-db": {
+    "info": {
+      "db-name": "demoservice"
+    },
+    "root-pw": "{{wizard_mysql_password_root}}",
+    "create-db" : {
+      "db-collision": "skip"
+    },
+    "grant-user": {
+      "user-name" : "demoservice",
+      "rand-pw" : true
```

Comment:
This is not implemented unfortunately

Reply:
Did this work in DSM 6 ?

Reply:
> 
> 
> Did this work in DSM 6 ?

Yes it works, and the random password is written into `demoservice.ini` in this demo package at installation time.

Reply:
👍